File: Sound_Card.pl
my ($signal, $sign);

$signal = <>;
$sign = <>;
chomp $signal;

if (($signal >= -1) && ($signal <= 1)) {
  say $signal;
} else {
  if ($signal > 1) {
    say 1;
  } else {
    say -1;
  }
}

print $sign;

# by Tovarisch Trunaev, 2025, Russia, Uray
# $Om->Delf();

File: Sound_Card.pas
Program Sound_Card;
Var 
    signal: real;
    sign: string;
Begin;
    ReadLn (signal);
    RealLn (sign);

    If (signal >= -1 AND signal <= 1) Then
      WriteLn(signal);
    else if (signal > 1) Then
        WriteLn(1);
      else
        WriteLn(-1);
    WriteLn (sign);
end.

{ by Tovarisch Trunaev, 2025, Russia, Uray }
{ Om->Delf(); }
